arbitrary-file-write/file-append/README.md

```diff
@@ -2,4 +2,49 @@
 * as root or user you can append data to a file and ...
 	* fully control content
 	* partially control content (eg. parts of a logfile)
-	* none (empty file)
\ No newline at end of file
+	* none (empty file)
+
+### sshd
+Looking at a mostly-default sshd config file most things are commented out so anything you append will take precedence if there is not something already before it (TODO: add sources for order of parsing).  
+
+#### Clean file ending
+```
+# Allow client to pass locale environment variables
+AcceptEnv LANG LC_*
+
+# override default of no subsystems
+Subsystem	sftp	/usr/lib/openssh/sftp-server
+
+# Example of overriding settings on a per-user basis
+#Match User anoncvs
+#	X11Forwarding no
+#	AllowTcpForwarding no
+#	PermitTTY no
+#	ForceCommand cvs server
+```
+
+#### Messy file ending (user specific and all users)
```

Comment:
I'm looking at the clean file and the messy file but I cannot tell what is "messy" about it. does it allow binary data? How does it react to just random text? 

The messy file should contain all the possible garbage that still keep it valid